### PR TITLE
Docs Update

### PR DESCRIPTION
**Pull request by Yefenyi**
Correcting incorrect links and sentence in developerGuide
Update image

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# DoTomorrow - Developer Guide
+# AddressBook Level 4 - Developer Guide
```
Comment (JoshuaChing) [inline docs/DeveloperGuide.md]: why is this changed?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
 * [Appendix E : Product Survey](#appendix-e--product-survey)
 
 
-## 1. Setting up
+<h2 name="setting-up">1. Setting up</h2>
```
Comment (evanyeung) [inline docs/DeveloperGuide.md:20]: Do you need a name attribute on these? Can you just leave them as is? The `##` in markdown will be rendered as an `h2` when displayed

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -73,7 +73,7 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
 * Solution: [Run tests using Gradle](UsingGradle.md) once (to refresh the libraries).
 
 
-## 2. Design
+<h2 name="design">2. Design</h2>
```
Comment (evanyeung) [inline docs/DeveloperGuide.md:76]: Same here

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -212,9 +212,9 @@ The `Storage` component,
 
 ### 2.6. Common classes
 
-Classes used by multiple components are in the `seedu.addressbook.commons` package.
+Classes used by multiple components are in the `seedu.task.commons` package.
 
-## 3. Implementation
+<h2 name="implementation"> 3. Implementation</h2>
```
Comment (evanyeung) [inline docs/DeveloperGuide.md:217]: ^^

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -266,7 +264,7 @@ Storing commands means there is no duplicated data stored. With the current impl
 
 The "snapshot" method was chosen over the "command" method largely because of its reduced complexity and increased robustness. Building reverse commands introduces a more complex system that increases the chance of bugs. Building snapshots of the tasks is also unlikely to overload memory as tasks are fairly small. If the data held by a task ever becomes much larger, this decision should be revisited.
 
-## 4. Testing
+<h2 name="testing">4. Testing</h2>
```
Comment (evanyeung) [inline docs/DeveloperGuide.md:267]: ^^

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -313,7 +311,7 @@ Thanks to the [TestFX](https://github.com/TestFX/TestFX) library we use,
    [here](http://stackoverflow.com/questions/2522897/eclipse-junit-ea-vm-option). <br>
    Delete run configurations created when you ran tests earlier.
 
-## 5. Dev Ops
+<h2 name="dev-ops">5. Dev Ops<h2>
```
Comment (evanyeung) [inline docs/DeveloperGuide.md:314]: ^^

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -422,12 +420,13 @@ Use case ends.
...
 
-> Use case ends
+ Use case ends
 
 3a. The given index is invalid
 
-> 3a1. DoTomorrow shows an error message <br>
-  Use case resumes at step 2
+> 3a1. DoTomorrow shows an error message
+
```
Comment (evanyeung) [inline docs/DeveloperGuide.md:428]: Is there a reason you're adding a free line rather than the `<br>`? This doesn't actually render to the same thing. There will be more spacing between the lined this way. If you don't like the `<br>` tag, you can end a line with multiple spaces to do the same thing. But we seem to be using `<br>` tags throughout.

**Review by evanyeung — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (JoshuaChing) [reply docs/DeveloperGuide.md:20]: I think this was to fix the anchoring. @Yefenyi can you confirm?

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Yefenyi) [reply docs/DeveloperGuide.md]: I don't know why it is changed, it just happens every time in my push...

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (Yefenyi) [reply docs/DeveloperGuide.md:20]: I tried to use markdown links only but those lines are started with number, it seems that then can not be properly linked to.

Comment (evanyeung) [reply docs/DeveloperGuide.md:20]: Okay, that's fine

**Review by evanyeung — APPROVED** [review]
📝 📝 📝

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (JoshuaChing) [inline docs/DeveloperGuide.md]: please fix

Comment (JoshuaChing) [inline docs/DeveloperGuide.md]: please fix

**Review by JoshuaChing — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Yefenyi) [reply docs/DeveloperGuide.md]: I have changed it back

**Review by JoshuaChing — APPROVED** [review]
